tests/run-make/foreign-exceptions/rmake.rs

```diff
@@ -0,0 +1,18 @@
+// This test was created to check that compilation and execution still works
+// after the addition of a new feature, in #65646: the ability to unwind panics
+// and exceptions back and forth between Rust and C++. Should this feature be broken,
+// the test should fail.
```

Comment:
Suggestion: reword as basic smoke test, this test is not guaranteed to fail if the feature is broken in subtle ways.